### DISCUSSION TITLE
How to see the log in Azure DevOps pipeline after PowerPlatformImport failure?

### DISCUSSION THREAD

**Discussion by autohub12** [microsoft/powerplatform-build-tools#737] [Q&A] [answered]
I received the following messages during PowerPlatformImportData step in Azure Devops pipeline

How can I see the detail log file? 

[ 'Connected as ***' ]
[ 'Importing data ...' ]
[ '    Parsing import data file' ]
[ '    Parse Complete' ]
[ '    Beginning Import To CRM.. Importing 14 entities - 743 records' ]
[ '    Running Schema Validation' ]
[
  '    Schema Validation failed for Missing fields on Entities, See log for missing fields'
]
[ '    Import Process completed' ]
[ '' ]
[ 'Data import failed. See messages above and log files for details.' ]
[
  'Action Result: Connected as ***,Importing data ...,    Parsing import data file,    Parse Complete,    Beginning Import To CRM.. Importing 14 entities - 743 records,    Running Schema Validation,    Schema Validation failed for Missing fields on Entities, See log for missing fields,    Import Process completed,,Data import failed. See messages above and log files for details.'
]
[ 'Authentication profiles and token cache removed' ]
[
  'The Clear Authentication Result: Authentication profiles and token cache removed'
]
Finishing: PowerPlatformImportData

**Comment by petrochuk** [comment, marked as answer]
you can select "enable diagnostics" option when you run your pipeline and it will output more detailed logging
